repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/entity/FieldTypeListItemEntity.java

```diff
@@ -3,15 +3,16 @@
 import javax.persistence.*;
 import java.io.Serializable;
 
-import static javax.persistence.GenerationType.IDENTITY;
+import static javax.persistence.GenerationType.SEQUENCE;
 
 @Table(name = "field_type_list_item")
 @Entity
 public class FieldTypeListItemEntity implements Serializable {
 
     @Id
     @Column(name = "id")
-    @GeneratedValue(strategy = IDENTITY)
+    @GeneratedValue(strategy = SEQUENCE, generator = "field_type_list_item_id_seq")
```

Comment:
if you are introducing sequences on the table now, what happens to the existing values (identities). what is going to be the start value of the new sequence!!!
this applies to all sequences above!!!

Reply:
These are not new sequences, the existing sequences that were already used to generate the ids are being altered so the existing sequence values will be preserved.